ACE fails with missing install target

https://travis-ci.org/CCPPETMR/SIRF-SuperBuild/jobs/400142749 failed with at the end
```
-- ACE install command succeeded.  See also /home/travis/build/CCPPETMR/SIRF-SuperBuild/builds/ACE/build/stamp/ACE-install-*.log
[100%] Completed 'ACE'
[100%] Built target ACE
[ 90%] Performing install step for 'ACE'
make[3]: *** No rule to make target `install'.  Stop.
make[2]: *** [builds/ACE/stamp/ACE-install] Error 2
make[1]: *** [CMakeFiles/ACE.dir/all] Error 2
```

--- a/SuperBuild/External_ACE.cmake
+++ b/SuperBuild/External_ACE.cmake
@@ -60,14 +60,14 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
     DOWNLOAD_DIR ${${proj}_DOWNLOAD_DIR}
     STAMP_DIR ${${proj}_STAMP_DIR}
     TMP_DIR ${${proj}_TMP_DIR}
+    INSTALL_DIR ${libACE_Install_Dir}
 	
+    INSTALL_COMMAND make ACE 
     CMAKE_ARGS
-        -DCMAKE_INSTALL_PREFIX=${Gadgetron_Install_Dir}
-	    INSTALL_DIR ${Gadgetron_Install_Dir}
+        -DCMAKE_INSTALL_PREFIX=${libACE_Install_Dir}
     DEPENDS
         ${${proj}_DEPENDENCIES}
   )
-
     set(${proj}_ROOT        ${${proj}_SOURCE_DIR})
     set(${proj}_INCLUDE_DIR ${${proj}_SOURCE_DIR})
 

